controllers/kafka.go

```diff
@@ -0,0 +1,49 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+	"os"
+	"zestream/utils"
+
+	"github.com/gin-gonic/gin"
+)
+
+/*
+	{
+	  "video": {
+	    "id": "string",
+	    "src": "string",
+	    "type": "mp4"
+	  }
+	}
+*/
+type Video struct {
+	ID   string `json:"id"`
+	Src  string `json:"src"`
+	Type string `json:"type"`
+}
+
+type Body struct {
+	Video Video `json:"video"`
+}
+
+func PublishMessage(c *gin.Context) {
+	var request Body
+	if err := c.ShouldBindJSON(&request); err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+	jsonBytes, err := json.Marshal(request)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+	var message = utils.PublishMessage(os.Getenv("KAFKA_URI"), string(jsonBytes), "video")
```

Comment:
could you create a config file which contains a map, where values like these are initialised and import the map and use value like this
`config[KAFKA_URL]`